docs/plugins.rst

```diff
@@ -127,6 +130,9 @@ For example,
 
 * For ``Operator`` plugin, an ``execute`` method is compulsory.
 * For ``Sensor`` plugin, a ``poke`` method returning a Boolean value is compulsory.
+* For ``AirflowPluginDagSource`` plugin, ``name``, ``add_dags_to_dagbag``, and
```

Comment:
```suggestion
* For ::class:`~airflow.plugin.AirflowPluginDagSource` plugin, ``name``, ``add_dags_to_dagbag``, and
```
This will allow automatic linking to the class documentation when such documentation will be created.

Reply:
I gotta fix the test cases and then I'll crack on with the doc problems :-D